services/common/load.go

```diff
@@ -605,6 +606,16 @@ var (
 			Value:       10,
 			Destination: &cfg.CacheSnapshot,
 		},
+		&cli.BoolFlag{
+			Name:        "autocollectevm",
+			Usage:       "auto collect miner coinbase utxo to evm",
+			Destination: &cfg.AutoCollectEvm,
+		},
+		&cli.StringFlag{
```

Comment:
I don't think it's safe to use a password as the node parameter.  If the use case is truly necessary, please consider to use a password file instead of a plaintext password. For me, it's extremely dangerous, better to avoid to allow the use case.